views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -90,6 +88,9 @@ define([
 
             //attach the element to the menu area
             var $container = this.getAreaBroker().getMenuArea();
+            if (this.getHost().getItem().bdy.bdy === "") {
+                this.disable();
```

Comment:
How will it be enabled again?

I mean, OK, you rendered a disabled previewer tool, but then you add an interaction into the test.
The button still remains disabled, which is an unexpected behavior IMO.

Reply:
Yes, I like that perfect solution, committed